lib/Agrammon/DB/Dataset.pm6

```diff
@@ -266,6 +266,7 @@ class Agrammon::DB::Dataset does Agrammon::DB {
              ORDER BY data_instance_order ASC, data_var
             DATASET
             $!data = $results.arrays;
+            $!comment
         }
         return self;
```

Comment:
This `$!comment` seems useless given we `return self` immediately below, so it cannot be the return value.

Reply:
Indeed, that's a left over from I don't know what ... removed.